### PR TITLE
[issue-6860] fix NPE of Could not generate api file for

### PR DESCRIPTION
**Pull request by shijinkui**
### PR checklist

fix NPE of Could not generate api file for. 

Add check type npe.

### CODE REVIEW

File: modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java [hunk 1]
```diff
@@ -2907,7 +2907,7 @@ protected String getOrGenerateOperationId(Operation operation, String path, Stri
      * @return true if the library/module/package of the corresponding type needs to be imported
      */
     protected boolean needToImport(String type) {
-        return !defaultIncludes.contains(type)
+        return type != null && type.length() > 0 && !defaultIncludes.contains(type)
```
Comment (wing328) [inline modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java]: @shijinkui Thanks for the PR. Please use `!StringUtils.isEmpty(type)` instead.

**Comment by shijinkui** [comment]
ping @wing328

**Comment by wing328** [comment]
@shijinkui quick question. Why use `isNoneEmpty` instead of `isNotEmpty`? Seems like `isNoneEmpty` is for multiple input: https://commons.apache.org/proper/commons-lang/apidocs/org/apache/commons/lang3/StringUtils.html#isNoneEmpty-java.lang.CharSequence...-

**Comment by shijinkui** [comment]
@wing328 Use `isNotBlank` instead.

**Comment by wing328** [comment]
@shijinkui ok. That will do. Will merge it after the CI jobs finish.

**Comment by wing328** [comment]
To close #6860

**Pull request merged**